### PR TITLE
Doc-1022 Update OpenAI node doc

### PR DESCRIPTION
**Pull request by ayatnkw**
[DOC-1022](https://linear.app/n8n/issue/DOC-1022/n8nn8n-nodes-langchainopenai)
- Add step-by-step instructions for each operation and action
- Separate doc by operation type

### CODE REVIEW

File: docs/integrations/builtin/app-nodes/n8n-nodes-langchain.openai/text-operations.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+	Refer to [Prompt engineering | OpenAI](https://platform.openai.com/docs/guides/prompt-engineering){:target=_blank .external-link} for more information on how to write a better prompt by utilizing these three roles. 
+- **Simplify Output**: Turn on to return a simplified version of the response instead of the raw data. 
+- **Output Content as JSON**: Turn on to attempt to return the response in JSON format. Compatible with `GPT-4 Turbo` and all `GPT-3.5 Turbo` models newer than `gpt-3.5-turbo-1106`.
+
+### Options
+
+- **Frequency Penalty**: Apply a penalty to reduce the model's tendency to repeat similar lines. The range is between `0.0` and `2.0`.
+- **Maximum Number of Tokens**: Set the maximum number of tokens for the response. One token is roughly four characters for standard English text. Use this to limit the length of the output. 
+- **Number of Completions**:
```
Comment (freakwriter) [inline docs/integrations/builtin/app-nodes/n8n-nodes-langchain.openai/text-operations.md:34]: @ayatnkw I'm guessing this one just got lost in the mix; would you be able to add a description for this option?

**Review by freakwriter — CHANGES_REQUESTED** [review]
@ayatnkw can you take a look at the one file comment I left here? I was able to resolve the other things I saw but thought that one might be faster for you to do.

File: docs/integrations/builtin/app-nodes/n8n-nodes-langchain.openai/text-operations.md (hunk 1 above)
Comment (ayatnkw) [reply docs/integrations/builtin/app-nodes/n8n-nodes-langchain.openai/text-operations.md:34]: sure!

**Comment by ayatnkw** [comment]
@freakwriter filled in the `number of completions` description. Would be great if you can take a final look.

**Review by nik8n — APPROVED** [review]
Really solid work Aya. We still have to add FAQ / common issues, but that's something we can do later.

**Review by freakwriter — APPROVED** [review]
Strong work on this, @ayatnkw . Good to go!

**Pull request merged**